Find a functionally equivalent code:android.provider.DocumentsContract.deleteDocument:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * {@hide}
 */

@@ -1,5 +1,5 @@
 {
     final Bundle in = new Bundle();
-    in.putString(Document.COLUMN_DOCUMENT_ID, getDocumentId(documentUri));
+    in.putParcelable(DocumentsContract.EXTRA_URI, documentUri);
     client.call(METHOD_DELETE_DOCUMENT, null, in);
 }

